pkg/data/management/podadmissionconfigurationtemplate_data.go

```diff
@@ -23,7 +23,7 @@ var FeatureAppNS = []string{
 	"cattle-neuvector-system",         // Neuvector
 	"cattle-monitoring-system",        // Monitoring and Sub-charts
 	"rancher-alerting-drivers",        // Alert Driver
-	"cis-operator-system",             // CIS Benchmark, RKE2 restricted PSA Config
+	"compliance-operator-system",      // Compliance Operator, RKE2 restricted PSA Config
```

Comment:
replacing this would mean any existing setup for cis would break.. better to add another entry for the namespace.. and remove the cis one in 2.13 or something..